images/virtualization-artifact/pkg/controller/cvmi_controller.go

```diff
@@ -17,60 +17,10 @@ limitations under the License.
 package controller
 
 import (
-	"context"
-
-	"github.com/go-logr/logr"
 	corev1 "k8s.io/api/core/v1"
-	"sigs.k8s.io/controller-runtime/pkg/controller"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-
-	"github.com/deckhouse/virtualization-controller/pkg/dvcr"
-	"github.com/deckhouse/virtualization-controller/pkg/sdk/framework/two_phase_reconciler"
 )
 
 const (
-	cvmiControllerName = "cvi-controller"
-	cvmiShortName      = "cvi"
-
 	ImporterPodVerbose    = "3"
 	ImporterPodPullPolicy = string(corev1.PullIfNotPresent)
```

Comment:
Do we need these constants and this file? These constants present in new controller: cvi_controller.go.

Reply:
Used by VI controller, should be removed in vi pull request